build.gradle

```diff
@@ -892,6 +927,7 @@ project(':connect:api') {
     testCompile libs.junit
 
     testRuntime libs.slf4jlog4j
+    testCompile project(':clients').sourceSets.test.output
```

Comment:
@ewencp, are you OK with this and similar changes to the connect test dependencies?